src/EditorFeatures/VisualBasicTest/CodeActions/IntroduceVariable/IntroduceVariableTests.vb

```diff
@@ -1171,7 +1171,6 @@ Imports System
 Module Program
     Sub Main()
         If True Then
-
```

Comment:
changed because i tweaked the formatter to not introduce a blank line with nested if-statements + elastic trivia.